packages/medusa/src/types/analytics-config.ts

```diff
@@ -0,0 +1,9 @@
+export type CreateAnalyticsConfig = {
+  opt_out: boolean
+  anonymize: boolean
+}
+
+export type UpdateAnalyticsConfig = {
+  opt_out: boolean
+  anonymize: boolean
```

Comment:
question: shouldn't these be optional since it's an update, and in the request to update the properties are optional?